README.md

```diff
@@ -7,18 +7,20 @@ The build information is sent to Artifactory in json format.
 
 ## Building and Testing the Sources
 
-The code is built using Gradle and includes integration tests.<br/>
-It must run using JDK 8 and Gradle 5.6.2. If you are using different gradle version you can use the provided gradle wrapper.<br/>
-In order to run tests the following environment variable must be provide:
+* The code is built using Gradle and includes integration tests.<br/>
+* It must run using JDK 8 and Gradle 5.6.2. If you are using different gradle version you can use the provided gradle wrapper.<br/>
+* In order to run tests the following environment variable must be provided:
```

Comment:
environment variable --> environment variables
must be provided: --> must be set:

Reply:
Fixed.